client/my-sites/posts/post-controls/index.jsx

```diff
@@ -167,25 +174,6 @@ const getAvailableControls = props => {
 	return controls;
 };
 
-const getControlElements = controls => controls.map( ( control, index ) =>
-	<li
-		className={ classNames( { 'post-controls__disabled': control.disabled } ) }
-		key={ index }
-	>
-		<a
-			className={ `post-controls__${ control.className }` }
-			href={ control.href }
-			onClick={ control.disabled ? noop : control.onClick }
-			target={ control.target ? control.target : null }
-		>
-			<Gridicon icon={ control.icon } size={ 18 } />
-			<span>
-				{ control.text }
-			</span>
-		</a>
-	</li>
-);
-
 export const PostControls = props => {
```

Comment:
Just thinking here, but I think this component would be easier to read if this was more declarative. The more button logic seems to be a blocker for that unless we can think of something clever.

A rough example:
```jsx
export function PostControls( { translate, canUserEditPost } ) {
     return (
         <div className={ classes }>
             <ul className={ ulClasses } >
                  <PostControl className='edit' href={ editURL } icon='pencil' onClick={ recordEditPost } >
                        translate( 'Edit' )
                  </PostControl>
                 //...
             </ul>
          </div>
      );
}
```